test/FileCheck/dump-input/fail.test

```diff
@@ -0,0 +1,5 @@
+RUN: not FileCheck -dump-input=fail --input-file=%S/Input/input.input %s  2>&1 | FileCheck  %S/Input/check-input.input 
```

Comment:
I think you have this test written backwards. The files in the `Inputs` should be the full test case, so they should be the input and check lines for the first `FileCheck` call. This file's check lines should be the ones verifying the result (the second FileCheck call).

Reply:
Fixed.